requirements.txt

```diff
@@ -27,7 +30,7 @@ jsonschema==2.5.1  # MIT
 lxml==3.6.0  # BSD
 nameparser==0.4.0  # LGPL
 newrelic==2.68.0.50  # newrelic APM agent, Custom License
-pendulum==0.6.2  # MIT
+pendulum==1.2.0  # MIT
```

Comment:
It was probably removed in version [1.0.0](https://pendulum.eustace.io/history/#1-0)

> [Removed deprecated methods](https://pendulum.eustace.io/history/#0-8).